Digipost/APIClient.swift

```diff
@@ -433,9 +381,15 @@ class APIClient : NSObject, NSURLSessionTaskDelegate, NSURLSessionDelegate, NSUR
                         self.deleteRefreshTokensAndLogoutUser()
                     }
             })
-        }else if (oAuthToken == nil) {
+        } else if (oAuthToken == nil) {
             assert(false," something wrong with oauth token")
         } else {
```

Comment:
For example if using scope idporten4 and refreshing list. It will retry with a lower level scope, "the current highest scope valid" which means 3, fullhighAuth or full, if any of those are present.

Or if you ask to read a letter with fullhighauth, the token is expired, and you retry request with a scope idporten4.